File: src/vb/file_system.vb
Imports System
Imports System.IO

Public Module Program
    Sub Main()
        Dim directoryInfo As DirectoryInfo = new DirectoryInfo("../files")

        For Each folder As DirectoryInfo In directoryInfo.GetDirectories()
            Console.WriteLine("/" + folder.Name)
        Next
        
        For Each file As FileInfo In directoryInfo.GetFiles()
            Console.WriteLine(file.Name + "," + file.Length.ToString() + "B")
        Next
    End Sub
End Module

File: src/cs/file_system.cs
using System;
using System.IO;

class Program
{
	public static void Main()
	{
        var directory = new DirectoryInfo("../files");

        foreach (var folder in directory.GetDirectories())
        {
            Console.WriteLine("/" + folder.Name);
        }

        foreach (var file in directory.GetFiles())
        {
            Console.WriteLine(file.Name + "," + file.Length);
        }
	}
}